promql/promqltest/test.go

```diff
@@ -376,6 +413,17 @@ func (t *test) parseEval(lines []string, i int) (int, *evalCmd, error) {
 			break
 		}
 
+		// This would still allow a metric named 'expect' if it is written as 'expect{}'.
```

Comment:
Is there a test for this case? I couldn't see one.

Reply:
I'll add a case for this.